java/modules/core/src/main/java/org/apache/synapse/config/xml/EntryFactory.java

```diff
@@ -83,9 +85,9 @@ public static Entry createEntry(OMElement elem, Properties properties) {
                 descriptionElem.detach();
             }
 
-            String src  = elem.getAttributeValue(
+            @SuppressWarnings("startswith") @StartsWith({"https", "file"}) String src  = elem.getAttributeValue(
                     new QName(XMLConfigConstants.NULL_NAMESPACE, "src"));
-
+            //TRUE POSITIVE: HTTP can creep in as URL is being loaded from the source
```

Comment:
From what source? Also "can creep in" probably shouldn't appear in technical writing, which true/false positive judgements are.